_summaries/sukhbaatar2015end-to-end_2.md

```diff
@@ -1,19 +1,42 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{sukhbaatar2015end-to-end}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: mmcinnestaylor # Your GitHub id
+score: 8 # How did you like this paper 0(dislike) to 10(love)
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Core Idea
+* End-To-End Memory Networks are a type of Memory Network with a possibly large external memory which is are trained end-to-end.
+* End-to-end training requires less supervision, thus improving applicability in real world settings.
+
+## Technical Implementation
+![Architecture](sukhbaatar2015end-to-end_2_b.png) 
+* The model accepts a discrete set of inputs (a story) along with a query, and outputs an answer.  
+![Q&A](sukhbaatar2015end-to-end_2_a.png)  
+* The inputs {x} and query q are transformed using embedding matrices A and B, respectively.  
+$$m_i = \text{Embed}_A(x_i)\qquad u = \text{Embed}_B(q)$$
+* The the most relevant embedded story input is matched with the embedded query by taking the inner product of each input with the query followed by a softmax operation, thus producing a probability vector.  
```

Comment:
Slight typo, "The the" -> "The"

Reply:
"embedded memory" instead of "embedded story input" may be more clear here, just because its not all too clear whats a memory vs whats a query and which of those are inputs etc.  You could also solve this ambiguity by explaining how information is fed into the model a bit.

Reply:
@Zayne-sprague I'm going to commit a change which tries to clarify these things without adding much more technical detail. I only hesitate to start adding information about the dimensions of all these vectors and matrices just to keep the summary brief. Let me know what you think!

Reply:
Yeah, this is perfect!  You could make a slight change from "The most relevant" to "The relevance of an embedded memory is..." since the operation will calculate the probability for each memory (just some have a higher p_i value) but I think this is a super small detail that could be ignored.  I think this change is great!

Reply:
I updated the language of that line to offer some more clarity.